geocoder_tester/world/france/iledefrance/test_addresses.csv

```diff
@@ -155,9 +155,9 @@ query,lon,lat,limit,expected_name,expected_housenumber,expected_street,expected_
 1 Rue de la Fontaine du Sault Héricy,,,,1 Rue de la Fontaine du Sault,,,,77850
 1 Rue de la Fontaine du Sault 77850,,,,1 Rue de la Fontaine du Sault,,,Héricy,
 1 Rue de la Fontaine du Sault,2.766645,48.449632,,1 Rue de la Fontaine du Sault,,,,77850
-14 Ruelle du Moulin Brûlé Jouy-sur-Morin,,,,14 Ruelle du Moulin Brûlé,,,,77320
-14 Ruelle du Moulin Brûlé 77320,,,,14 Ruelle du Moulin Brûlé,,,Jouy-sur-Morin,
-14 Ruelle du Moulin Brûlé,3.283984,48.794960,,14 Ruelle du Moulin Brûlé,,,,77320
+14 Ruelle du Moulin Brûlé Jouy-sur-Morin,,,,14 Ruelle du Moulin Brulé,,,,77320

```

Comment:
not since 1990 ;) 
https://fr.wikipedia.org/wiki/Rectifications_orthographiques_du_fran%C3%A7ais_en_1990#D.C3.A9tails